.github/scripts/test.sh

```diff
@@ -10,32 +12,26 @@ if [ "`uname -s`" == "Darwin" ] ; then
     fi
 fi
 
-# For now do not run tests with the documentation build
-if [ "${DOCS}" == true ]; then exit 0; fi
-
-# No test data, then remove submodule (git automatically clones recursively)
-if [ ${TEST} == none ];
- then git submodule deinit -f .;
-fi
-
-# Install test data as a package, then remove the submodule
-if [ ${TEST} == package ];
- then pip install ./sherpa-test-data;
- pip install pytest-xvfb;
- git submodule deinit -f .;
-fi
-
 # Build smoke test switches, to ensure requested dependencies are reachable
 if [ -n "${XSPECVER}" ]; then XSPECTEST="-x -d"; fi
 if [ -n "${FITS}" ] ; then FITSTEST="-f ${FITS}"; fi
 smokevars="${XSPECTEST} ${FITSTEST} -v 3"
 
-# Install coverage tooling and run tests using setuptools
+# Install coverage tooling and run tests
 if [ ${TEST} == submodule ]; then
-    # pip install pytest-cov codecov;
-    conda install -yq pytest-cov codecov;
-    python setup.py -q test -a "--cov sherpa --cov-report term" || exit 1;
-    codecov;
+    # With an install build we have trouble running the tests
+    # from the installation directory (unable to import sherpa.utils._utils)
+    # and it is not at all clear why this is, so work around it.
+    #
+    if [ ${ITYPE} == install ]; then
+      cd /home
+      # Possibly seeing issues with the coverage checks
+      # pytest --pyargs sherpa --cov sherpa || exit 1;
+      pytest --pyargs sherpa || exit 1;
```

Comment:
with my new changes, I don't think coverage being in a different place will be a problem. It corrects paths that are different than the installed. It'd be best to keep coverage checks in IMO.

Maybe drop most of this change?

Reply:
Yeah - I was waiting for #1339 to re-work this part!

Reply:
I do like how with #1339 and this work we can make the GH actions a bit more declarative (i.e. move the logic out of a bunch of bash if statements)!

Reply:
Awesome, I just wanted to note it to make it easier to find later. And yeah, I'm seeing the benefits of declarative configuration over "dynamic"/"programmatic" or whatever it's called.